### PR TITLE
Add unit testing infra to common_recipe 

### PR DESCRIPTION
**Pull request by tschuy**
This will allow us to write unit tests for the "common" LWRP before it gets merged into master.

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+# Style tests. Rubocop and Foodcritic
+namespace :style do
+  desc 'Run Ruby style checks'
+  RuboCop::RakeTask.new(:ruby)
+
+  desc 'Run Chef style checks'
+  FoodCritic::Rake::LintTask.new(:chef) do |t|
+    t.options = {
+      :fail_tags => ['any'],
```
Comment (iankronquist) [inline Rakefile]: what is this?

File: Rakefile [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      :tags => ['~FC005']
```
Comment (iankronquist) [inline Rakefile]: We probably don't want this.

File: Rakefile (hunk 1 above)
Comment (tschuy) [inline Rakefile]: fail_tags are the tags that it should fail on. Essentially, always fail for all fails.

**Comment by iankronquist** [comment]
Looks good to me.

**Comment by tschuy** [comment]
Merged into common_recipe so we can write tests.

**Pull request merged**